modules/nw-ingress-setting-update-subnet-LoadBalancerService.adoc

```diff
@@ -0,0 +1,80 @@
+// Modules included in the following assemblies:
+//
+// * ingress/configure-ingress-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-ingress-setting-update-subnet-LoadBalancerService_{context}"]
+= Updating the subnets on an existing Ingress Controller
+You can update an `IngressController` with manually specified load balancer subnets in {product-title} to avoid any disruptions, to maintain the stability of your services, and to ensure that your network configuration aligns with your specific requirements. The following procedures show you how to select and apply new subnets, verify the configuration changes, and confirm successful load balancer provisioning. 
```

Comment:
```suggestion
= Updating the subnets on an existing Ingress controller

You can update an Ingress controller with manually specified load balancer subnets in {product-title} to avoid any disruptions, to maintain the stability of your services, and to ensure that your network configuration aligns with your specific requirements. The following procedures show you how to select and apply new subnets, verify the configuration changes, and confirm successful load balancer provisioning. 
```

That first sentence is doing a lot :) can it be broken up?

We tend to want to avoid self-referential language, would it make sense to just remove the second sentence? Or rewrite as things the user can do? (i.e. "You can use the {oc-first} to select and apply new subnets, verify the configuration changes, and confirm successful load balancer provisioning." or similar)